src/components/note/note.pw.tsx

```diff
@@ -0,0 +1,252 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import {
+  NoteComponent,
+  NoteComponentWithInlineControl,
+  NoteComponentWithPreviews,
+} from "./components.test-pw";
+import Box from "../box";
+import { tooltipPreview, dlsRoot } from "../../../playwright/components/index";
+import { linkPreview } from "../../../playwright/components/link-preview/index";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import {
+  noteComponent,
+  noteHeader,
+  noteFooterCreatedBy,
+  noteFooterChangeTime,
+  noteStatus,
+  noteContent,
+} from "../../../playwright/components/note/index";
+import {
+  checkAccessibility,
+  getStyle,
+} from "../../../playwright/support/helper";
+import { actionPopoverWrapper } from "../../../playwright/components/action-popover/index";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+
+test.describe("check props for Note component", () => {
+  testData.forEach((text) => {
+    test(`should render Note with text prop set to ${text}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<NoteComponent text={text} />);
+
+      await expect(noteContent(page)).toHaveText(text);
+    });
+  });
+
+  [
+    { width: 30, widthInPx: 75 },
+    { width: 75, widthInPx: 187 },
+  ].forEach(({ width, widthInPx }) => {
+    test(`should render Note with width prop set to ${width}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <Box width="250px">
+          <NoteComponent width={width} />
+        </Box>
+      );
+
+      const widthVal = await getStyle(noteComponent(page), "width");
+      const value = parseInt(widthVal);
+
+      await expect(noteComponent(page)).toHaveAttribute("width", `${width}`);
+      expect(value).toBeGreaterThanOrEqual(widthInPx - 1);
+      expect(value).toBeLessThanOrEqual(widthInPx + 1);
+    });
+  });
+
+  test("should render Note with inlineControl prop", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<NoteComponentWithInlineControl />);
+
+    await expect(actionPopoverWrapper(page)).toBeVisible();
+  });
+
+  testData.forEach((title) => {
+    test(`should render Note with title prop set to ${title}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<NoteComponent title={title} />);
+
+      await expect(noteHeader(page)).toHaveText(title);
+    });
+  });
+
+  testData.forEach((name) => {
+    test(`should render Note with name prop set to ${name}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<NoteComponent name={name} />);
+
+      await expect(noteFooterCreatedBy(page)).toHaveText(name);
+    });
+  });
+
+  test("should render Note with createdDate prop", async ({ mount, page }) => {
+    const createdDate = "25 June 2022, 11:57 AM" as string;
```

Comment:
**suggestion(non-blocking)**: You shouldn't need the type assertion here, as TypeScript will infer `createdDate` as a string constant automatically for you.

```suggestion
    const createdDate = "25 June 2022, 11:57 AM";
```